src/convertToObject.js

```diff
@@ -11,8 +11,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  let stylesArray = sourceString.split(';');
+  const stylesObject = {};
+
+  stylesArray = stylesArray.map(item => item.trim());
+  stylesArray = stylesArray.filter(item => item !== '');
+  stylesArray = stylesArray.map(item => item.split(':'));
+  stylesArray = stylesArray.map(item => item.map(i => i.trim()));
```

Comment:
prepare you array inside stylesArray variable, no need to reassign it each time
like this

```
let stylesArray = sourceString
.split(';')
.map(...)
.filter(...)
```